docs/docs/best-practices/structuring-your-projects.md

```diff
@@ -0,0 +1,104 @@
+- **Services have different release cycles.** Separate projects allow teams to manage their flags without affecting other services.
+- **You need distinct access control.** With separate projects, you can grant teams full access to their own flags whilst restricting visibility of other projects.
+- **Features are truly independent.** If there's no need for flags to be shared or coordinated between services, separate projects reduce clutter.
+
+## When to use a single project
+
+Consider using a single project when:
+
+- **Services are tightly coupled.** If multiple services need to coordinate changes (such as a database schema migration), sharing flags in a single project makes this easier and safer.
+- **You share user context across services.** A single project allows you to define [Segments](/flagsmith-concepts/segments) once and use them across all services, rather than duplicating segment definitions.
+- **You want a unified view of your feature flags.** A single project gives you visibility of all active flags and their states in one place.
+- **You're a small team.** With fewer people and services, the simplicity of a single project often outweighs any organisational benefits of multiple projects.
+
+## Microservice architectures
+
+If you're running a microservice architecture, the decision depends on how coupled your services are.
+
+**Tightly coupled services** are good candidates for sharing a single Flagsmith project. For example, if multiple services share a database and you need to coordinate a schema change, using a single project with shared flags makes this much easier than coordinating deployments across separate projects.
+
+**Loosely coupled services** with well-defined interface boundaries are generally better suited to one project per service. If a service's API is stable, consuming services don't need to know about its internal feature flags.
+
+For more details on how project structure affects API usage, see [Micro-service Architectures](/best-practices/efficient-api-usage#micro-service-architectures).
+
+## Multiple teams
+
+When you have many teams, there are two main approaches:
+
+### One project per team
+
+This approach works well when:
+
+- Teams have clear ownership boundaries
+- Teams work independently with minimal coordination
+- You want to simplify access control (each team manages their own project)
+
+### Projects organised by domain
+
+This approach works well when:
+
+- Multiple teams contribute to the same service or domain
+- You need to coordinate flag changes across teams
+- Teams share common segments or user context
+
+In practice, many organisations use a hybrid approach: separate projects for truly independent services, but shared projects where teams need to collaborate.
+
+## Practical considerations
+
+### System limits
+
+Each project has [system limits](/administration-and-security/governance-and-compliance/system-limits) including:
+
+- 400 features per project
+- 100 segments per project
```

Comment:
Since these limits are likely to change in the not too distant future (and hopefully be entirely removed eventually), I'd rather not define the actual limits in multiple places in the docs. Can we just keep this to a reference to the existing page on system limits?